server/src/main/java/io/druid/server/coordinator/DruidCoordinator.java

```diff
@@ -233,6 +233,35 @@ public boolean isLeader()
 
   public CountingMap<String> getSegmentAvailability()
   {
+    return getSegmentStats(
+        new Function<DataSegment, Integer>()
+        {
+          @Override
+          public Integer apply(DataSegment input)
+          {
+            int available = (segmentReplicantLookup.getTotalReplicants(input.getIdentifier()) == 0) ? 0 : 1;
```

Comment:
Can you use boolean logic here and make it numeric only in the return value? using numbers as booleans makes things harder to read.